extensions/aria_extension_tosca/profiles/tosca-simple-1.0/interfaces.yaml

```diff
@@ -63,24 +63,48 @@ interface_types:
     pre_configure_source:
       description: >-
         Operation to pre-configure the source endpoint.
+      _extensions:
+        default_dependencies:
+          - edge > source
```

Comment:
I thought you said that if there is no cost to allowing the user to override it, we could.